numalogic/udfs/postprocess.py

```diff
@@ -137,11 +127,17 @@ def exec(self, keys: list[str], datum: Datum) -> Messages:
         #  Postprocess payload
         if payload.status in (Status.ARTIFACT_FOUND, Status.ARTIFACT_STALE) and thresh_artifact:
             try:
-                anomaly_scores = self.compute(
+                feature_scores = self.compute(
                     model=thresh_artifact.artifact,
                     input_=payload.get_data(),
+                    score_conf=_conf.numalogic_conf.score,
                     postproc_clf=postproc_clf,
+                )  # (nfeat,)
+
```

Comment:
Can we add `_update_info_metric`. Similar to like https://github.com/numaproj/numalogic/blob/780f2e23cccf3648b385499c589347c3bed0461e/numalogic/udfs/inference.py#L162C13-L162C32

Reply:
I will leave that one for you to take care . :)